lib/wickedbase.pm

```diff
@@ -233,9 +233,18 @@ sub ping_with_timeout {
     while ($timeout > 0) {
         if (script_run($ping_command) == 0) {
             if ($args{count_success} > 1) {
-                my $cnt = $args{count_success} - 1;
+                my $cnt       = $args{count_success} - 1;
+                my $threshold = 50;
                 $ping_command =~ s/\s-c\s1\s+/ -c $cnt /;
-                validate_script_output($ping_command, sub { /\s+0% packet loss/ });
+                my $ping_out = script_output($ping_command, proceed_on_failure => 1);
```

Comment:
if we force proceed_on_failure => 1 here, we don't need it as  input parameter (line 226).

Reply:
we do need it later , see line 245 ( old version ) / 254 ( new version )

Reply:
It's used in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7600/files#diff-7c6669e0bb0ed1f0eed4a62dc81f200dR254